x-pack/plugins/observability_solution/apm/public/assistant_functions/get_apm_timeseries.tsx

```diff
@@ -110,12 +110,12 @@ export function registerGetApmTimeseriesFunction({
 
                     case 'transaction_latency':
                       if (
-                        series.stat.timeseries.function ===
+                        series.stat.timeseries.latencyAggType ===
                         LatencyAggregationType.p99
                       ) {
                         chartType = ChartType.LATENCY_P99;
                       } else if (
-                        series.stat.timeseries.function ===
+                        series.stat.timeseries.latencyAggType ===
```

Comment:
breaking change 😄 you'll have to take this into account in the render function, so not sure if it's worth it

Reply:
Ok, not worth it then